stories/healthselectorv2.stories.js

```diff
@@ -0,0 +1,70 @@
+//@flow
+import React from 'react';
+import styled from 'styled-components';
+import Healthselector from '../src/lib/components/healthselectorv2/HealthSelector.component';
+import { Wrapper, Title } from './common';
+import Icon from '../src/lib/components/icon/Icon.component';
+
+export default {
+  title: 'Components/v2/Healthselector',
+  component: Healthselector,
+};
+
+const HealthSelectorWrapper = styled.div`
+  width: 180px;
+`;
+
+export const Default = () => {
+  return (
+    <Wrapper>
+      <Title>Basic usage</Title>
+      <HealthSelectorWrapper>
+        <Healthselector
+          onChange={(value) => {
+            console.log(value);
+          }}
+        />
+      </HealthSelectorWrapper>
+
+      <Title>changing labels</Title>
+      <HealthSelectorWrapper>
+        <Healthselector
+          onChange={(value) => {
+            console.log(value);
+          }}
+          label="alerts"
+          optionsConfiguration={{
+            healthy: {
+              label: 'coucou',
+              shortLabel: 'wesh',
+            },
+            all: {
+              label: 'patrick',
+              shortLabel: 'late',
+            },
+            warning: {
+              label: 'attention',
+              shortLabel: 'ouch',
+              icon: <Icon name="Tape" />,
+              value: 'myValue',
```

Comment:
Please avoid french and strange values in your test cases. Ideally having business oriented test cases is always better